### PR TITLE
Fix 日志分享页报错

### PR DESCRIPTION
**Pull request by Carlmac**
### Description

Fixes # (issue)
修复日志分享页出现多余报错的问题

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard-front/src/http/fetch/error-interceptor.ts [hunk 1]
```diff
@@ -3,7 +3,16 @@ import { Message } from 'bkui-vue';
...
+import { useCommon } from '@/store';
+import { nextTick } from 'vue';
+
 const { BK_LOGIN_URL } = window;
+let common: ReturnType<typeof useCommon> | null = null;
+
+// 用 nextTick 包裹避免在根 app 装载 pinia 前调用 store 发生的错误
+nextTick(() => {
+  common = useCommon();
+});
```
Comment (ielgnaw) [inline src/dashboard-front/src/http/fetch/error-interceptor.ts]: 写在拦截器方法里，可以直接用 `useCommon()`

File: src/dashboard-front/src/views/operate-data/access-log/detail.vue [hunk 2]
```diff
@@ -14,7 +14,12 @@
             <div class="item" v-for="({ label, field }, index) in details.fields" :key="index">
               <dt class="label">{{label}}</dt>
               <dd class="value">
-                {{field === 'timestamp' ? transformTime(details.result[field]) : details.result[field]}}
+                {{ field === 'timestamp'
+                  ? transformTime(details.result[field])
+                  : details.result[field] === false
+                    ? false
+                    : details.result[field] || '--'
```
Comment (ielgnaw) [inline src/dashboard-front/src/views/operate-data/access-log/detail.vue]: 多层的三元运算符有点乱

File: src/dashboard-front/src/http/fetch/error-interceptor.ts (hunk 1 above)
Comment (Carlmac) [reply src/dashboard-front/src/http/fetch/error-interceptor.ts]: 已修改 ✔

File: src/dashboard-front/src/views/operate-data/access-log/detail.vue (hunk 2 above)
Comment (Carlmac) [reply src/dashboard-front/src/views/operate-data/access-log/detail.vue]: 已修改 ✔

**Review by ielgnaw — APPROVED** [review]

**Pull request merged**